ton_sdk/Cargo.toml

```diff
@@ -42,6 +42,7 @@ optional = true
 
 [dependencies.ton_executor]
 git = "ssh://git@github.com/tonlabs/ton-executor.git"
+branch = "feature/use-atomic"
```

Comment:
SDK is not complied because of private repos dependencies in ton-executor. Merge SDK after fixing this.
And don't forget to remove this line